### PR TITLE
Make code review on survey form project

### PR DESCRIPTION
**Pull request by rosemutai**
On this project:

- I added a survey form
- I added CSS styles for form styling
- I configured HTML & CSS  linters

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  nodechecker:
+    name: node_modules checker
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check node_modules existence
+        run: |
+          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
```
Comment (rosemutai) [inline .github/workflows/linters.yml:56]: Great job! All Linters passed 💯

File: index.html [hunk 2]
```diff
@@ -1,13 +1,76 @@
...
 </head>
+
 <body>
-  
+  <div class="main-content">
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form action="" id="survey-form">
+
+      <div class="name">
```
Comment (rosemutai) [inline index.html:18]: Use semantic HTML tags. For example, here I would advise you to use the <main></main> tag. These will improve the web accessibility of your app.

File: index.html [hunk 3]
```diff
@@ -1,13 +1,76 @@
...
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Survey Form</title>
   <link rel="stylesheet" href="index.css">
 </head>
+
 <body>
-  
+  <div class="main-content">
+    <h1 id="title">freeCodeCamp Survey Form</h1>
```
Comment (rosemutai) [inline index.html:14]: Use semantic HTML tags. For example, here I would advise you to use the <header></header> tag. These will improve the web accessibility of your app.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- ACKNOWLEDGEMENTS -->
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+I would like to thank Microverse for this exercise.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/rosemutai/My-Portfolio/blob/main/LICENSE) licensed.
```
Comment (rosemutai) [inline README.md:197]: Please add a license file you have referred to here. This will enable anyone reading your code to reference it here.

File: .github/workflows/linters.yml (hunk 1 above)
Comment (bobb-Rob) [reply .github/workflows/linters.yml:56]: Good job giving very warm kudos. 👏🏾

File: index.html (hunk 3 above)
Comment (bobb-Rob) [reply index.html:14]: - [x] Well done on making the review comment. It would have been much better if your review comment is clear on what should be done. Also, it sounds more like a command or order. Usually, when giving code reviews, it is expected that your points be clear and easy to understand and that it is polite. 

> Use semantic HTML tags. For example, here I would advise you to use the tag. These will improve the web accessibility of your app.

The first sentence is ambiguous, and not clear. The question is **_what tag are you talking about?_.** Kindly state clearly what and why it should be done. **_Your comment above was good in that it gave a reason for the suggestion._**

File: index.html (hunk 2 above)
Comment (bobb-Rob) [reply index.html:18]: - [x] Hi @rosemutai, I still see the same unclear comment. Kindly rephrase it and make it **_polite_** and with **_clear_** meaning passed across. 😄

File: README.md (hunk 4 above)
Comment (bobb-Rob) [reply README.md:197]: You can rephrase this to give the student a guide on how to add this file.  E.g 👇🏾 

> Kindly add a valid LICENSE. You can follow the instructions on the right side of the article attached below to set up a valid LICENSE - https://choosealicense.com/licenses/mit/

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @rosemutai, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -1,13 +1,76 @@
...
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
```
Comment (rosemutai) [inline index.html:15]: Kindly make sure you use the header tag to enclose the title and the description. This helps to make your markup more semantic.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (rosemutai) [inline README.md:200]: Kindly add a valid LICENSE. You can follow the instructions on the right side of the article attached below to set up a valid LICENSE - https://choosealicense.com/licenses/mit/

**Review by amejid — APPROVED** [review]
Hi @rosemutai, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Good use of Github features ✔️ 
- Comments are polite and provide a reason ✔️ 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
